packages/plugins/rpc-manager/src/lib/eth_sendTransaction.ts

```diff
@@ -67,7 +63,7 @@ export default class EthSendTransaction extends RpcModifier {
 
     try {
       // Check if we have that account in our wallet
-      const account = accounts.find((acc) => Web3.utils.toChecksumAddress(acc.address) === Web3.utils.toChecksumAddress(params.request.params[0].from));
+      const account = this.accounts.find((acc) => Web3.utils.toChecksumAddress(acc.address) === Web3.utils.toChecksumAddress(params.request.params[0].from));
```

Comment:
Should it be `acc.address || acc` as seen in [other lines](https://github.com/embark-framework/embark/pull/2150/files#diff-7f2d476e5b66f22cdc294475317d84f9R33)?

Reply:
Have removed `acc.address || acc` from the other source as it was not needed.